src/MSALWrapper/AuthFlow/AuthFlowExecutor.cs

```diff
@@ -51,7 +51,7 @@ public async Task<AuthFlowResult> GetTokenAsync()
                 if (attempt == null)
                 {
                     var oopsMessage = $"Auth flow '{authFlow.GetType().Name}' returned a null AuthFlowResult.";
-                    result.Errors.Add(new Exception(oopsMessage));
+                    result.Errors.Add(new NullTokenResultException(oopsMessage));

```

Comment:
>NullTokenResultException [](http://example.com/codeflow?start=42&length=24)

While we are here - do we know how often we are seeing a null authflowresult failure right now? If we are seeing this error in telemetry - are we able to understand why we are seeing them? do we have enough data to understand the scenario or do we need to log more information here to make it more useful?

Reply:
I haven't explored telemetry enough to know whether we see these. @mvanchaa, do you have any insight here? 🤔

Reply:
I will take a look at it

Reply:
Looks like we never got this issue. I see zero errors in this scenario.

Reply:
That's surprising to me. Does that seem right to you, @kyle-rader? 🤔

Reply:
Yeah, ideally this exception should never happen.

Reply:
I'm also happy to approve this PR and move this discussion to a different forum.